kong/resty/dns/client.lua

```diff
@@ -1045,15 +1045,9 @@ end
 local function search_iter(qname, qtype)
   local _, dots = qname:gsub("%.", "")
 
-  local type_list, type_start, type_end
-  if qtype then
-    type_list = { qtype }
-    type_start = 0
-  else
-    type_list = typeOrder
-    type_start = 0   -- just start at the beginning
-  end
-  type_end = #type_list
+  local type_list = qtype and { qtype } or typeOrder
+  local type_start = 0
+  local type_end = #type_list
```

Comment:
Should we keep the comment `-- just start at the beginning`?

Reply:
It's unnecessary because it's self-explanatory.